tests/stdlib/tstrutil.nim

```diff
@@ -7,6 +7,11 @@ discard """
 import
   strutils
 
+template rejectParse(e) =
```

Comment:
This just ignores the expression if it throws a value error. Shouldn't it fail if a value error isn't raised?

Reply:
Haha, indeed. Thanks! I even added a test case, that's obviously not going to fail (checking for "0bFF" with `parseHexInt`).